tests/json/bitcoin-config-test.json

```diff
@@ -1,10 +1,16 @@
 {
+  "bitcoinFeeSpendingCutoffPeriodInBlocks": 1,
+  "bitcoinFeeSpendingCutoff": 0.01,
   "bitcoinPeerUri": "unused",
-  "bitcoinWalletImportString": "cVneRge89Va3mrxv7dehTYTknbgLVan8rY5hkbXUwdwXtZrrueGY",
+  "bitcoinRpcUsername": "",
+  "bitcoinRpcPassword": "",
+  "bitcoinWalletOrImportString": "cVneRge89Va3mrxv7dehTYTknbgLVan8rY5hkbXUwdwXtZrrueGY",
```

Comment:
We don't want to check this in right?

Reply:
It's a dummy import string so that the initialization won't throw error, I could spy and mock, but this was easier at the time.